<!DOCTYPE HTML>
<!--
	Iridium by TEMPLATED
    templated.co @templatedco
    Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Danko Lab</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<link href='http://fonts.googleapis.com/css?family=Arimo:400,700' rel='stylesheet' type='text/css'>
		<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
		<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-panels.min.js"></script>
		<script src="js/init.js"></script>
		<noscript>
			<link rel="stylesheet" href="css/skel-noscript.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="css/ie/v9.css" /><![endif]-->
	</head>
	<body class="no-sidebar">

		<!-- Header -->
		<div id="header">
			<div class="container"> 
				
				<!-- Logo -->
				<div id="logo">
					<h1><a href="#">Danko Lab</a></h1>
					<!--<span>Website coming soon!</span>-->
				</div>
				
				<!-- Nav -->
				<nav id="nav">
					<ul>
						<li><a href="index.html">Homepage</a></li>
						<li class="active"><a href="Research.html">Research</a></li>
						<li><a href="Team.html">Team</a></li>
						<li><a href="Publications.html">Publications</a></li>
						<li><a href="Software.html">Software</a></li>
						<li><a href="Contact.html">Find Us</a></li>
					</ul>
				</nav>
			</div>
		</div>

		<!-- Main -->
		<div id="main">
			<div class="container">
				<div class="row"> 
					

					<!-- Content -->
					<div id="content" class="7u skel-cell-important">
						<section>

							<header>
								<h2>Research</h2>
								<!-- <span class="byline">Research in the Danko Lab</span> -->
							</header>
							<br>
							<p>The Danko lab studies how our DNA sequences control complex programs of gene transcription. Our work is primarily focused on understanding how natural genetic differences between species affect the various steps in the RNA polymerase II transcription cycle. Our work provides insight into the molecular basis behind phenotypic differences between species. In addition, studying the millions of naturally occurring random genetic mutation “experiments” also presents an exciting opportunity to understand the fundamental principles by which our DNA sequences encode gene expression. <br><br>
							We are an interdisciplinary research group making extensive use of both computational and molecular tools. Our specialty is developing statistical and machine-learning approaches to analyze functional genomic sequencing data prepared using Hi-C, ATAC-seq, PRO-seq, RNA-seq, and related assays. Our tools borrow a variety of ideas from the fields of statistics and machine-learning, including recent uses of hidden Markov models, support vector machines, and artificial neural networks. We also run an active wet-lab that has made great strides in developing and using run-on and sequencing technologies to map the location of RNA polymerase, including <a title="[Mahat et. al., PRO-seq Protocol]" href="http://www.nature.com/nprot/journal/v11/n8/full/nprot.2016.086.html">PRO-seq</a> and <a title="[Chu et. al., ChRO-seq paper]" href="http://www.biorxiv.org/content/early/2017/09/07/185991">ChRO-seq</a>. More recently we have begun using Hi-C/ Hi-ChIP, single cell RNA-seq, and CRISPR epigenome editing technologies.
							</p>

							<BR>
							<header><span class="subsectit">How does gene expression contribute to phenotypic differences between species?</span></header>
							<p class="text"><BR>

							Understanding the chain of molecular events that link DNA sequence ‘genotype’ to organism ‘phenotype’ is one of the most exciting frontiers in modern genetics. DNA sequences located in non-coding regions of the genome are critical drivers of phenotypic differences, both between and within species. <br><br>
							Our objective is to discover the fundamental rules by which transcriptional changes arise from differences in DNA sequence and chromatin packaging within the nucleus. To achieve this goal we integrate genomic data collected using a combination of molecular assays (PRO-seq, RNA-seq, ATAC-seq, and Hi-C). Most of our work focuses on CD4+ T-cells, a lynchpin in the adaptive immune system undergoing rapid evolutionary changes that are relevant to autoimmune and allergic disorders. <br><br>
							Our <a title="[Danko et. al., Evolution in ensembles of enhancers]" href="https://www.nature.com/articles/s41559-017-0447-5">most recent work</a> has found that although changes in distal regulatory elements arise rapidly, these changes frequently do not lead to measurable differences in the transcription of nearby genes. We found evidence that gene transcription is stabilized by multiple compensatory changes acting across ensembles of distal enhancers. This finding suggests a model of regulatory evolution in which changes in regulatory activities arise rapidly, and gene expression is held constant through widespread compensation between regulatory elements targeting each gene.
							</p>

							<BR>
							<header><span class="subsectit">Developing RNA polymerase as a mark denoting multiple ‘layers’ of genome function.</span></header>
							<p class="text"><BR>
							
							Detecting biochemically active DNA sequences in a cell (one of the <a title="[Pisco et. al., Conceptual Confusion: the case of Epigenetics]" href="https://www.biorxiv.org/content/early/2016/05/12/053009">common definitions of a cell's “epigenome”</a>) is a major challenge in genomics.  Many approaches rely on using dozens of separate experimental assays, making the analysis of new cell systems expensive and time-consuming. We have recently demonstrated that RNA polymerase marks a surprisingly <a title="[Core, Martins et. al., PRO-cap]" href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3142.html">broad</a> <a title="[Andersson, Sandelin , Danko. Unified Model.]" href="http://www.sciencedirect.com/science/article/pii/S0168952515001043">variety</a> of functional elements across the genome. These functional elements can be recognized based on their characteristic “shapes” extracted from PRO-seq data using <a title="[Danko et. al., dREG paper]" href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3329.html">machine learning tools</a>. <br><br>
							Our objective is to develop a computational toolkit that deconvolves a single PRO-seq assay into a rich source of information about multiple ‘layers’ of functional elements that are active in our genomes. We have developed a machine learning tool called <a title="[Danko et. al., dREG paper]" href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3329.html">dREG</a> which identifies the location of active regulatory DNA sequence elements using PRO-seq data as input. More recently, we have trained discriminative support vector machines to identify transcription factor binding sites with a high accuracy, and we have begun using transcription to guess or ‘impute’ the abundance of covalent modifications to core histones. These technologies allow comprehensive annotation of active functional elements in mammalian genomes using PRO-seq data alone.<br><br>
							Finally, a core mission of the wet-lab is to extend run-on and sequencing assays to map the location of RNA polymerase across the genome in a wider range of biological conditions.  We have recently introduced a new run-on and sequencing variant called <a title="[Chu et. al., ChRO-seq paper]" href="http://www.biorxiv.org/content/early/2017/09/07/185991">ChRO-seq</a> to address the key problem with PRO-seq: namely that it requires a nuclear isolation, which can be challenging in complex tissue samples such as muscle or brain.  We have also made substantial progress on strategies to multiplex the PRO-seq and ChRO-seq assays using a 96-well plate format. Taken together, our efforts significantly expand the scope and range of applications in which PRO-seq can be applied.
							</p>

							<BR><BR>
							<header><span class="subsectit">Core papers from the Danko lab.</span></header>
							<p class="text">
							<BR>

							<strong>Identification of regulatory elements from nascent transcription using dREG.</strong> <br>
								<span class="cgd">Wang Z</span>, <span class="cgd">Chu T</span>, <span class="cgd">Choate LA</span>, <span class="cgd">Danko CG.</span><br>
								<a title="[Genome Research]" href="https://genome.cshlp.org/content/29/2/293.short">Genome Research</a> (2019).
							<BR><BR>

							<strong>Chromatin run-on and sequencing maps the transcriptional regulatory landscape of glioblastoma multiforme.</strong> <br>
							<span class="cgd">Chu T</span>, <span class="cgd">Rice EJ</span>, Booth GT, Salamanca HH, <span class="cgd">Wang Z</span>, Core LJ, Longo SL, Corona RJ, Chin LS, List JT, Kwak H, <span class="cgd">Danko CG</span>.<br>
							<a title="[Nature Genetics]" href="https://www.nature.com/articles/s41588-018-0244-3">Nature Genetics</a> (2018).
							<BR><BR>

							<strong>Dynamic evolution of regulatory element ensembles in primate CD4+ T cells.</strong> <br>
							<span class="cgd">Danko CG</span>, <span class="cgd">Choate LA</span>, Marks BA, <span class="cgd">Rice EJ</span>, <span class="cgd">Wang Z</span>, <span class="cgd">Chu T</span>, Martins AL, Dukler N, Coonrod SA, Tait-Wojno E, List JT, Kraus WL, Siepel A. <br>
							<a title="[Full text]" href="https://www.nature.com/articles/s41559-017-0447-5">Nature Ecology & Evolution</a> (2018).
							<BR><BR>

							<strong>A unified architecture of transcriptional regulatory elements.</strong> <br>
							Andersson R, Sandelin A, <span class="cgd">Danko CG</span>.<br>
							<a title="[Full text]" href="http://www.sciencedirect.com/science/article/pii/S0168952515001043">Trends in Genetics</a> (2015).
							<BR><BR>

							<strong>Identification of active transcriptional regulatory elements from GRO-seq data. </strong> <br>
							<span class="cgd">Danko CG</span>, Hyland SL, Core LJ, Martins AL, Waters CT, Lee HW, Cheung VG, Kraus WL, Lis JT, and Siepel A. <br>
							<a title="[Full text]" href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3329.html">Nature Methods</a>  (2015). <br>
				
							</p>
							
					</section>
					</div>

					<!-- Sidebar -->
					<div id="sidebar" class="5u">
						<section>
							<ul class="style">
								<li>
									<strong>Danko lab - Baker Institute Research Highlight</strong>
									<iframe width="420" height="315" allowfullscreen="allowfullscreen"
									src="https://www.youtube.com/embed/OpFOJfGNb_A">
									</iframe>
								</li>

								<li>
									<p class="posted">October 2018</p>

									<img src="images/ChROseq-Paper-Fig1.png" width="420" alt="" />
									<p class="text"><a title="[Nature Genetics]" href="https://www.nature.com/articles/s41588-018-0244-3">Chromatin run-on and sequencing maps the transcriptional regulatory landscape of glioblastoma multiforme.</a> Introducing ChRO-seq and leChRO-seq: New nascent transcription assays used to analyze 24 primary glioblastoma tissue samples.</p> 
									<strong>Highlights:</strong><br>
									* ChRO-seq can analyze virtually any frozen tissue sample.<br>
									* Enhancers identified in primary GBMs resemble open chromatin in the normal human brain.<br>
									* Three types of rare enhancers activated in tumors: stem, immune, and differentiated.<br>
									* Hundreds of transcription factors contribute to heterogeneity between GBMs.<br>
									* Transcription factor activity predicts clinical outcomes.<br>
									<br>
									<strong>Charles talks about ChRO-seq - CSHL Transcription (2017)</strong>
									<iframe width="420" height="315" allowfullscreen="allowfullscreen"
									src="https://www.youtube.com/embed/UJDarHI48Iw">
									</iframe>
								</li>

								<li>
									<p class="posted">January 2018</p>
									<img src="images/NHP-Paper-Redundancy.png" width="420" alt="" />
									<p class="text"><a title="[Nature Eco & Evo]" href="https://www.nature.com/articles/s41559-017-0447-5">Dynamic evolution of regulatory element ensembles in primate CD4+ T cells.</a> Our research paper on the molecular mechanisms behind evolutionary changes in transcription, focusing on CD4+ T-cells in primates.</p> 
									<strong>Highlights:</strong><br>
									* Evolutionary changes at enhancers correlate with distance to genes and chromatin interactions.<br>
									* Enhancers/ promoters spread across a locus undergo selective pressures as a unit, or an "ensemble".<br>
									* Changes often preserve transcriptional output at genes.<br>
									* Experiments confirm multiple substitutions that contribute to human-specific transcription of SGPP2.<br>
								</li>

								<li>
									<p class="posted">August 2015</p>
									<img src="images/UnifiedArchitecture.png" width="420" alt="" />
									<p class="text">Opinion paper arguing for a <a title="[Trends in Genetics]" href="http://www.sciencedirect.com/science/article/pii/S0168952515001043">unified model of regulatory elements</a>.</p> 
									<strong>Highlights:</strong><br>
									* Transcription marks both enhancers and promoters.<br>
									* Some gene promtoers enhance transcription at distal sites.<br>
									* The function of a "regulatory element" is context dependent.<br>
								</li>
							</ul>
						</section>
					</div>

				</div>
			</div>
		</div>


		<!-- Footer -->
		<div id="footer">
			<div class="container">
				<div class="row">
					<div class="4u">
						<section>
							<h2>Danko Lab</h2>
							<ul class="contact">
								<li>
									<span class="mail">E-Mail</span>
									<span>
									<script type="text/javascript" language="Javascript">// <![CDATA[Hide script from non-javascript-enabled browsers
										var username="dankoc";
										var at="@";
										var domain="gmail.com";
										document.write("<a href='mailto:" +username+""+at+""+domain+ "'>" + username+""+at+""+domain + "</a>");
									</script>
									</span>
								</li>
								<li>
									<span class="mail">Social</span>
									<a href="https://twitter.com/charlesdanko"><img src="images/twitter.png" alt="" /></a>
									<a href="https://github.com/danko-lab"><img src="images/github.png" alt="" /></a>
								</li>
								<li>
									<span class="address">Address</span>
									<span>
										The Baker Institute for Animal Health<br />
										College of Veterinary Medicine <br />
										Cornell University <br />
										Hungerford Hill Road<br />
										Ithaca, NY 14853
									</span>
								</li>
							</ul>
						</section>
					</div>
					<div class="4u">
						<section>
							<h2>Cornell University</h2>
							<ul class="default">
								<li><a href="https://www2.vet.cornell.edu/departments-centers-and-institutes/baker-institute">The Baker Institute</a></li>
								<li><a href="http://www.vet.cornell.edu/biosci/">Department of Biomedical Sciences</a></li>
								<li><a href="http://www.vet.cornell.edu/">College of Veterinary Medicine</a></li>
								<li><a href="http://www.cb.cornell.edu/">Computational Biology Graduate Program</a></li>
								<li><a href="http://www.cornell.edu">Cornell University</a></li>
							</ul>
						</section>
					</div>
					<div class="4u">
						<section>
							<h2>Join the lab!</h2>
									<ul class="contact">
										<li>
										<span>We are considering applications from exceptionally talented scientists!</span>
										</li>
										<li>
										<span class="mail">Postdocs:</span>
										<span>E-mail a 1 page cover letter, a curriculum vitae, and 3 letters of reference to Charles Danko.</span>
										</li>
										<li>
										<span class="mail">Students:</span>
										<span>Apply through the graduate fields of <a href="http://www.cb.cornell.edu/">Computational Biology</a> or <a href="http://www.gendev.cornell.edu/">Genetics, Genomics & Development</a>.</span>
										</li>
									</ul>	
						</section>
					</div>
				</div>
			</div>
		</div>

		<!-- Copyright -->
		<div id="copyright">
			<div class="container">
				Design: <a href="http://templated.co">TEMPLATED</a>
			</div>
		</div>
		
	</body>
</html>
