
Detection of Regulatory Elements (Promoters + Enhancers) using GRO-seq or PRO-seq (dREG)

<p>

dREG is the result of a collaboration between the <a href="http://www.dankolab.org/">Danko</a>, <a href="http://compgen.bscb.cornell.edu/">Siepel</a>, and <a href="http://mbg.cornell.edu/research/lis-lab/">Lis</a> labs.

<p>

Read our preprint on <a href="http://www.biorxiv.org/content/early/2014/11/12/011353">bioRxiv</a>.

Abstract:
Identification of the genomic regions that regulate transcription remains an important open problem. We have recently shown that global run-on and sequencing (GRO-seq) with enrichment for 5-prime-capped RNAs reveals patterns of divergent transcription that accurately mark active transcriptional regulatory elements (TREs), including enhancers and promoters. Here, we demonstrate that active TREs can be identified with comparable accuracy by applying sensitive machine-learning methods to standard GRO-seq and PRO-seq data, allowing TREs to be assayed together with transcription levels, elongation rates, and other transcriptional features, in a single experiment. Our method, called discriminative Regulatory Element detection from GRO-seq (dREG), summarizes GRO-seq read counts at multiple scales and uses support vector regression to predict active TREs. The predicted TREs are strongly enriched for marks associated with functional elements, including H3K27ac, transcription factor binding sites, eQTLs, and GWAS-associated SNPs. Using dREG, we survey TREs in eight cell types and provide new insights into global patterns of TRE assembly and function. 

<p>

Get the dREG software here on <a href="https://github.com/Danko-Lab/dREG">GitHub</a>.

<p>

See dREG in the <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&hubUrl=http://charlesdanko.com/hub/dreg/hub.txt">UCSC genome browser</a> here: 
http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&hubUrl=http://charlesdanko.com/hub/dreg/hub.txt

<p>

NOTE: These browser tracks are a work in progress.